tests/commands/convert/test_input_console_app.py

```diff
@@ -0,0 +1,10 @@
+def test_overrides_are_used_for_console(convert_command):
+    overrides = {"console_app": "Console"}
+    out = convert_command.build_gui_context({}, overrides)
+    assert out["console_app"] == "true"
+
+
+def test_overrides_are_used_for_GUI(convert_command):
+    overrides = {"console_app": "GUI"}
+    out = convert_command.build_gui_context({}, overrides)
+    assert out["console_app"] == "false"
```

Comment:
There should also be a test of the two "normal" cases, where user input is provided.